libraries/script-engine/src/WebSocketClass.cpp

```diff
@@ -34,10 +34,11 @@ WebSocketClass::WebSocketClass(QScriptEngine* engine, QWebSocket* qWebSocket) :
 void WebSocketClass::initialize() {
     connect(_webSocket, &QWebSocket::disconnected, this, &WebSocketClass::handleOnClose);
     connect(_webSocket, &QWebSocket::textMessageReceived, this, &WebSocketClass::handleOnMessage);
+    connect(_webSocket, &QWebSocket::binaryMessageReceived, this, &WebSocketClass::handleOnBinaryMessage);
     connect(_webSocket, &QWebSocket::connected, this, &WebSocketClass::handleOnOpen);
     connect(_webSocket, static_cast<void(QWebSocket::*)(QAbstractSocket::SocketError)>(&QWebSocket::error), this,
         &WebSocketClass::handleOnError);
-    _binaryType = QStringLiteral("blob");
+    _binaryType = QStringLiteral("arraybuffer");
```

Comment:
This doesn't actually do anything because `_binaryType` isn't used. It would be good to sometime support `"blob"` vs. `"arraybuffer"` binary result types but need not be done now.